deploy/hypershift-cluster-monitoring-config/cluster-monitoring-config.yaml

```
@@ -0,0 +1,118 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: hshift-cluster-monitoring-config
```

```suggestion
  name: hypershift-cluster-monitoring-config
```

---

@supreeth7 The configmap name should be same the the classic one, isn't it?

---

Yes, that's right, apparently CMO defaults to recognizing configmap names "cluster-monitoring-config" only.
```suggestion
  name: cluster-monitoring-config
```